examples/reminders/lib/feature_reminder_list/ui_components/reminder_list_view.dart

```diff
@@ -6,30 +6,27 @@ import '../../base/models/reminder/reminder_model.dart';
 import '../blocs/reminder_list_bloc.dart';
 import 'scroll_to_index_widget.dart';
 
-class ReminderListView extends StatefulWidget {
+class ReminderListView extends StatelessWidget {
   const ReminderListView({
     this.createdReminderId,
     Key? key,
   }) : super(key: key);
   final String? createdReminderId;
 
-  @override
-  State<ReminderListView> createState() => _ReminderListViewState();
-}
-
-class _ReminderListViewState extends State<ReminderListView> {
   @override
   Widget build(BuildContext context) => RxPaginatedBuilder<ReminderListBlocType,
           ReminderModel>.withRefreshIndicator(
         state: (bloc) => bloc.states.paginatedList,
-        onBottomScrolled: (bloc) => bloc.events.loadPage(),
+        onBottomScrolled: (bloc) {
+          print('TTTTTT');
```

Comment:
Please make sure you remove the prints before you file a PR

Reply:
The print was removed